### ISSUE TITLE
Change key for Ctrl-Shift-MouseOver

### ISSUE THREAD

**Issue by pschimmel** [snoopwpf/snoopwpf#408]
Hi!
It's more a question than an issue.
Is it possible to change the keys with which I can use the selection feature that is associated to Ctrl-Shift-MouseOver by default?

**Comment by batzen** [comment]
It's currently hard coded at https://github.com/snoopwpf/snoopwpf/blob/6d805ca74ef5e8b7fbe35cf6cc5f02ae2e13b853/Snoop.Core/Windows/SnoopUI.xaml.cs#L638

Shouldn't be difficult to change it.
What's your use case for a different combination?

**Comment by pschimmel** [comment]
It's a personal issue. The control key of my laptop that I use for programming is broken ;)

**Comment by batzen** [comment]
Both control keys?

**Comment by pschimmel** [comment]
Yes. But I ordered a replacement keyboard in the meantime. Maybe changing the default keys could be useful when the same combination is used by the app being snooped.Am 15.05.2023 19:08 schrieb Bastian Schmidt ***@***.***>:
Both control keys?
